Simplenote.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,17 @@
 	objects = {
 
 /* Begin PBXAggregateTarget section */
+		3F1BB4AD243199FF006D1A04 /* ScreenshotsSecrets */ = {
+			isa = PBXAggregateTarget;
+			buildConfigurationList = 3F1BB4B4243199FF006D1A04 /* Build configuration list for PBXAggregateTarget "ScreenshotsSecrets" */;
+			buildPhases = (
+				3F1BB4B524319A04006D1A04 /* ShellScript */,
+			);
+			dependencies = (
+			);
+			name = ScreenshotsSecrets;
+			productName = ScreenshotsSecrets;
+		};
```

Comment:
I added the logic to copy the secret `ScreenshotsCredentials.swift` as dedicated aggregate target, rather than appending it to the one already in place for `SPCredentials.swift`. It seemed more appropriate to isolate this piece of work to the only target that needs it.